include/mbedtls/config.h

```diff
@@ -3671,6 +3671,25 @@
  */
 //#define MBEDTLS_PSA_HMAC_DRBG_MD_TYPE MBEDTLS_MD_SHA256
 
+/** \def MBEDTLS_PSA_MAX_KEY_BITS
+ * Restrict the PSA Library to supporting a maximum key size (in bits).
+ *
+ * If this option is unset, the library will fall back to a default value of
+ * 8191 bytes = 65528 bits.
+ */
+//#define MBEDTLS_PSA_MAX_KEY_BITS 0xfff8
+
+/** \def MBEDTLS_PSA_KEY_SLOT_COUNT
```

Comment:
Minor: the file is easier to maintain and to read through when options are sorted, so `MBEDTLS_PSA_KEY` should come before `MBEDTLS_PSA_MAX`.

Reply:
Fixed